### ISSUE TITLE
Question about Axial-Res50

### ISSUE THREAD

**Issue by lxa9867** [csrhddlam/axial-deeplab#27]
Thank you for sharing this great work. I have a question. Is Axial-Res50 same to the pretrained backbone you used in MaX-deeplab-S? When I use the provided code to pretrain it on Imagenet, it didn't reach the claimed number in your paper. What can I do to improve it? Thank you.

**Comment by csrhddlam** [comment]
Thanks for your interest in our new work MaX-DeepLab.

1. No. Axial-Res50 is not MaX-deeplab-S backbone. Please see MaX-DeepLab Appendix figure A.9 for more the architecture details.
2. 77% is definitely lower than expected. Please double check if the architecture and the number of parameters match those in the MaX-DeepLab paper. I would expect around 79% and 80% accuracy for MaX-deeplab-S and L (the pretrained part) at a 224x224 resolution. One might get higher numbers if one applies strong augmentation, label smoothing or super long training schedule, but these are not used in our pre-training.
3. We are working on releasing the code for MaX-DeepLab as well. Please stay tuned.

**Comment by lxa9867** [comment]
Thank you for your reply. I found bugs in my pretraining code while some details are still hard for me to reimplement like drop path in backbone. If you already have a codebase for MaX-Deeplab, I am more than happy to help you train the models.
